nncf/common/hardware/configs/template.json

```diff
@@ -16,14 +16,19 @@
                 "granularity": [
                     "pertensor",
                     "perchannel"
-                ]
+                ],
+                /*
+                 * Narrow range: should NNCF use 2**num_bits quants or 2**num_bits - 1
+                 */
+                "narrow_range": false
             },
             "q8_sym_tnr_-128_127": { // Alias name for set of hyperparameters
                 "bits": 8, // Number of quantization bits
                 "mode": "symmetric", // Quantization mode
                 "granularity": "pertensor", // Granularity: one scale for output tensor
                 "level_low": -128, // Low quantization level
```

Comment:
Does NNCF support for `level_low` and `level_high`?

Reply:
Isn't it obsolete, since other params define how to calculate them?

Reply:
https://github.com/openvinotoolkit/nncf/blob/develop/nncf/common/hardware/config.py#L160

Reply:
> https://github.com/openvinotoolkit/nncf/blob/develop/nncf/common/hardware/config.py#L160

It looks like, `level_low` and `level_high` parameters are only used to calculate `signedness_to_force`. I believe, that is more simple just specify `signedness_to_force` in the hardware config. I would suggest to implement it in this PR.

Reply:
Done